src/index.js

```diff
@@ -178,8 +178,19 @@ async function getSourceFileExtensions() {
   return sourceExts;
 }
 
+function fixPackages(cmd) {
```

Comment:
There seems to be a typo here. When testing, it throws an error that cwd is not defined.

Please ensure you have tested code before creating a PR. You can test CLIs by running `npm link` in your fork's directory. This will create a symlink, so that you can test it using regular vue-native-cli commands using the terminal.